composite/synth-index/README.md

```diff
@@ -0,0 +1,123 @@
+# Chainlink External Adapter to Synthetix Index Value
+
+The adapter calculates a Synthetix Index value in the currency selected
+
+## Configuration
+
+- `DEFAULT_NETWORK` (Optional). Network to fetch the Synth Index
+
+This adapter relies on [`token-allocation`](../../token-allocation/README.md) adapter. Required `token-allocation` input params and configuration apply to this adapter as well.
+
+## Input Params
+
+- `asset`, `from`: Synthx Index asset to fetch
```

Comment:
Add `base` here as a default param name. It's standard on our other pricing adapters. We also price an asset here, just the asset is more complex.